### PR TITLE
Improved readme file

### PR DESCRIPTION
**Pull request by ivsandeep**
Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # hw01
+# An amazing project
```
Comment (ptaylor1021) [inline README.md:2]: Hi Sandy! Like you, I wrote a short message. I hope you enjoy this quarter!

**Review by ptaylor1021 — COMMENTED** [review]
Hi Sandy! Like you, I wrote a short message. I hope you enjoy this quarter!

File: README.md (hunk 1 above)
Comment (ennissean) [inline README.md:2]: Good job, Sandy. This was short, sweet, and to the point. I think you could have spiced up your work a little bit by changing # An amazing project to # _An amazing project_

Comment (ennissean) [reply README.md:2]: And I could've improved my comment by using ### _An amazing project_

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 # hw01
```
Comment (majocamacho) [inline README.md:1]: Hi Sandy! I reviewed your homework and I feel I would have love if you include more about yourself and use other of the tools we saw in classes. When I did my homework, I started with headers and bold as you did. After I tried ordered lists (using *) and links (underlining the link). Next time I am planning to take more chances and include images or graphics. I am sure you will try new things over the time of the course!

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |     x         |            |
| **Presentation**            |           |              |      x      |
| **Achievement, creativity** |           |              |      x      |
| **Reproducibility**         |           |         x    |            |

* Note the instructions asked you to use at least 4 different Markdown elements. You showed you can follow the homework workflow, but not how to use Markdown to format text. You'll need to do this for upcoming assignments

**Pull request closed without merging**